### ISSUE TITLE
VPhysics Jolt

### ISSUE THREAD

**Issue by nn-firc** [SwagSoftware/Kisak-Strike#38]
https://github.com/Joshua-Ashton/VPhysics-Jolt

this

**Comment by nn-firc** [comment]
> > https://github.com/Joshua-Ashton/VPhysics-Jolt
> > 
> > this
> 
> Implement it yourself

I've already did that

**Comment by GuestSneezeOSDev** [comment]
> > > https://github.com/Joshua-Ashton/VPhysics-Jolt
> > > this
> > 
> > 
> > Implement it yourself
> 
> I've already did that

then why tf do yu need to keep this issue open 💀

**Comment by nn-firc** [comment]
> > > > https://github.com/Joshua-Ashton/VPhysics-Jolt
> > > > this
> > > 
> > > 
> > > Implement it yourself
> > 
> > I've already did that
> 
> then why tf do yu need to keep this issue open 💀 

not in this repo, kisak-strike it's almost dead

**Comment by GuestSneezeOSDev** [comment]
> > > > > https://github.com/Joshua-Ashton/VPhysics-Jolt
> > > > > this
> > > > 
> > > > 
> > > > Implement it yourself
> > > 
> > > 
> > > I've already did that
> > 
> > 
> > then why tf do yu need to keep this issue open 💀
> 
> not in this repo, kisak-strike it's almost dead

ok but do u have dc? can i contact you?

**Comment by LWSS** [comment]
Yeah it is dead, there aren't a lot of reasons to invest any serious effort into Valve's proprietary codebase unless you have some sort of  mod-project in mind.

You know, I personally like idtech4. It's better in a lot of ways. Everyone's got their own preference.

**Comment by GuestSneezeOSDev** [comment]
> Yeah it is dead, there aren't a lot of reasons to invest any serious effort into Valve's proprietary codebase unless you have some sort of mod-project in mind.
> 
> You know, I personally like idtech4. It's better in a lot of ways. Everyone's got their own preference.

If the repository is dead perhaps archive the repository.

**Comment by LWSS** [comment]
> > Yeah it is dead, there aren't a lot of reasons to invest any serious effort into Valve's proprietary codebase unless you have some sort of mod-project in mind.
> > You know, I personally like idtech4. It's better in a lot of ways. Everyone's got their own preference.
> 
> If the repository is dead then maybe archive the repository?

Or just look at the last commit date. I'm not opposed to merges, but I'm not working on it anytime soon

**Comment by GuestSneezeOSDev** [comment]
> > > Yeah it is dead, there aren't a lot of reasons to invest any serious effort into Valve's proprietary codebase unless you have some sort of mod-project in mind.
> > > You know, I personally like idtech4. It's better in a lot of ways. Everyone's got their own preference.
> > 
> > 
> > If the repository is dead then maybe archive the repository?
> 
> Or just look at the last commit date. I'm not opposed to merges, but I'm not working on it anytime soon

alr

**Comment by nn-fric** [comment]
> > > Yeah it is dead, there aren't a lot of reasons to invest any serious effort into Valve's proprietary codebase unless you have some sort of mod-project in mind.
> > > You know, I personally like idtech4. It's better in a lot of ways. Everyone's got their own preference.
> > 
> > If the repository is dead then maybe archive the repository?
> 
> Or just look at the last commit date. I'm not opposed to merges, but I'm not working on it anytime soon

probably i'll make a pr later they got most of code for csgo already so it's a drop-in
